lib/Dialect/CMakeLists.txt

```diff
@@ -4,5 +4,6 @@ add_subdirectory(FIRRTL)
 add_subdirectory(Handshake)
```

Comment:
We're not fully consistent here, but please add the standard license header boilerplate to these build files plz.